applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -185,12 +197,11 @@ public static ConfigurationClientBuilder builder() {
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * <pre>
-     * ConfigurationSetting setting = new ConfigurationSetting().key("prodDBConnection")
-     *     .label("westUS")
-     *     .value("db_connection");
+     * <p>Update the setting with the key-label pair "prodDBConnection"-"westUS" to have the value "updated_db_connection".</p>
      *
-     * ConfigurationSetting result = client.updateSetting(setting);
+     * <pre>
+     * ConfigurationSetting result = client
+     *     .updateSetting(new ConfigurationSetting().key("prodDBConnection").label("westUS").value("db_connection"));
```

Comment:
Same issue here with value not matching the text in the description. Feels like copy/paste fails :)

Reply:
Yes, yes there were a lot of those...